new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -140,16 +127,23 @@
    * @param label       a Map of key value pairs the service is decorated with
    * @param namespace   in which the service is running
    * @return true if the service exists otherwise false
-   * @throws ApiException when query fails
    */
-  public static Callable<Boolean> serviceExists(
-      String serviceName,
-      Map<String, String> label,
-      String namespace
-  ) throws ApiException {
-    return () -> {
-      return Kubernetes.doesServiceExist(serviceName, label, namespace);
-    };
+  public static Callable<Boolean> serviceExists(String serviceName, Map<String, String> label, String namespace) {
```

Comment:
curious - did you change this to satisfy checkstyle rules?

Reply:
yes